.github/workflows/main.yml

```diff
@@ -36,7 +36,8 @@ jobs:
           npm ci
           npm run format:check
 
-  get_docker_image_tags:
+  get_and_save_docker_image_tags:
+    if: github.event.inputs.updateImageTags == 'true'
```

Comment:
We don't want the `get_and_save_docker_image_tags` to run when there aren't new image tags.
- If we run it when the latest image tags haven't changed, this job will fail 
- Because the commit line will exit with an error if there's no diff to commit
- `git commit -m "Update image tags used in dev environment [skip ci]`

So we only want this to be run when we know there is a new latest image tag, which is when the `webapp` or `service` triggers the dispatch event with the `updateImageTags = 'true'` input.